suite/smoke/keeper_test.go

```diff
@@ -81,6 +81,7 @@ var _ = Describe("Keeper suite @keeper", func() {
 					FallbackLinkPrice:    big.NewInt(2e18),
 				},
 				1,
+				uint32(2500000), //upkeepGasLimit
```

Comment:
I'm just wondering, is there a maximum gas limit that a user can set when registering their upkeep? If there is, it might be worth making it a constant and exposing it to other packages.

Reply:
It's dependent on the maxPerformGas set in registry config which can be set differently in every test

Reply:
I see, thanks!